glean-core/src/lib.rs

```diff
@@ -114,6 +121,14 @@ impl Glean {
         Ok(())
     }
 
+    /// Called when Glean is initialized to the point where it can correctly
+    /// assemble pings. Usually called from the language specific layer after all
+    /// of the core metrics have been set and the ping types have been
+    /// registered.
+    pub fn on_ready_to_send_pings(&self) {
+        self.event_data_store.on_ready_to_send_pings(&self);
```

Comment:
Bikeshed:

I wonder whether we eventually have other things to call in here.
IMO then the `event_data_store.on_ready_to_send_pings` name can change to something more clearly identifying what the data store is doing there and the `on_ready_to_send_pings` is only the public name bundling it all

Reply:
That's not a bad suggestion.  I suppose the thing would be called `flush_events_from_previous_run`?